modules/installation-osp-converting-ignition-resources.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-user.adoc
+
+[id="installation-osp-converting-ignition-resources_{context}"]
+= Preparing the bootstrap Ignition file
+
+The bootstrap Ignition file that the installation program generates is too large to pass to {rh-openstack} directly.
+
+First, upload the bootstrap Ignition configuration file. Then, create a small Ignition file ("shim") that downloads the generated ignition file to Nova as user data.
```

Comment:
upload where? 
maybe say something like:
"First, upload the boostrap Ingnition configure file to an object storage service like swift or glance. Then create a small json file, or shim file, which will tell the cluster how to download the bootsrap Ignition configuration file form the service."

The "shim" file is downloaded to the nova servers as user data. The bootstrap Ignition files is downloaded from the objects storage service, or where ever it was uploaded to.